src/screens/CitySelectionScreen.tsx

```diff
@@ -0,0 +1,186 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { Alert } from 'react-native';
+
+import {
+  Button,
+  DropdownSelect,
+  EmptyMessage,
+  HtmlView,
+  LoadingSpinner,
+  SafeAreaViewFlex,
+  ServiceTiles,
+  Title,
+  Touchable,
+  Wrapper,
+  WrapperRow
+} from '../components';
+import { colors, Icon, normalize } from '../config';
+import { addToStore, readFromStore, removeFromStore } from '../helpers';
+import { useStaticContent } from '../hooks';
+import { DropdownProps } from '../types';
+
+const SELECTED_CITY = 'selectedCity';
+
+const useCitySelection = () => {
+  const [storedCity, setStoredCity] = useState<string | null>(null);
+  const [loading, setLoading] = useState<boolean>(true);
+
+  const loadStoredCity = useCallback(async () => {
+    setLoading(true);
+    const city = await readFromStore(SELECTED_CITY);
+    setStoredCity(city);
+    setLoading(false);
+
+    return city;
+  }, []);
+
+  const storeCity = useCallback(async (city: string | null) => {
+    if (city) {
+      await addToStore(SELECTED_CITY, city);
+      setStoredCity(city);
+    }
+  }, []);
+
+  const resetCity = useCallback(async () => {
+    await removeFromStore(SELECTED_CITY);
+    setStoredCity(null);
+  }, []);
+
+  return { storeCity, storedCity, loadStoredCity, loading, resetCity };
+};
+
+export const CitySelectionScreen = () => {
+  const { data: citiesData, loading: citiesLoading } = useStaticContent<Array<string>>({
+    refreshTimeKey: 'cities',
+    name: 'cities',
+    type: 'json'
+  });
+
+  const { data: htmlContent, loading: htmlLoading } = useStaticContent({
+    refreshTimeKey: 'cityHtmlContent',
+    name: 'cityHtmlContent',
+    type: 'html'
+  });
+
+  const initiallySelectedItem = {
+    id: 0,
+    index: 0,
+    value: 'z.B. Stuttgart',
+    selected: true
+  };
+  const [dropdownData, setDropdownData] = useState<DropdownProps[]>([initiallySelectedItem]);
+
+  const [selectedCity, setSelectedCity] = useState<string | null>(null);
+  const [contentName, setContentName] = useState<string | null>(null);
+  const { storeCity, loadStoredCity, storedCity, loading, resetCity } = useCitySelection();
+
+  useEffect(() => {
+    const setContentNameWithCity = async () => {
+      const city = await loadStoredCity();
+      if (city) {
+        setContentName(`city${city}`);
```

Comment:
why do we need the prefix `city`? and wouldn't it be more correct to separate the value from the prefix with a `-` for example like `city-Augsburg`? and could it be clever to lowercase the city value to get `city-augsburg`? and hat about city names with space, shouldn't they get kebab-cased like `city-linz-am-rhein`?